package.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "scripts": {
     "prepare": "npm run build",
-    "build": "tsc -p . -d",
+    "build": "tsc -p . -d && cp src/brotli-decode.* dist/src",
```

Comment:
does this serve as a replacement for https://github.com/soundcloud/intervene/blob/master/Makefile#L8? should we favor running the npm script directly over the make target?